wp/plugin/lib/utils.ml

```diff
@@ -14,6 +14,7 @@
 open !Core_kernel
 open Bap_main
 open Bap.Std
+open Regular.Std
```

Comment:
How did we gain this?

Reply:
I opened Regular (http://binaryanalysisplatform.github.io/bap/api/odoc/regular/Regular/Std/index.html) so I can use `Data.Cache.Digest.pp` which I added to the info logs.